.github/CODEOWNERS

```diff
@@ -938,6 +938,13 @@
 # ServiceLabel: %Astronomer %Mgmt
 # ServiceOwners:                                                   @banggaurav
 
+# PRLabel: %NewRelicObservability
+/sdk/newrelicobservability/Azure.ResourceManager.NewRelicObservability/       @dipeshbhakat-microsoft @vipray-ms
```

Comment:
dipeshbhakat-microsoft is missing the required public Azure org membership.

Reply:
Done